packages/react-components/src/components/knowledge-graph/KnowledgeGraphPanel.tsx

```diff
@@ -70,7 +171,44 @@ const KnowledgeGraphContainer: React.FC<KnowledgeGraphInterface> = ({ kgDataSour
           </Button>
         </SpaceBetween>
         {/* inline styling here for testing only this will be fixed in the next PR */}
-        <Graph elements={elements} style={{ width: '1000px', height: '1000px' }} />
+        <div ref={containerRef} className={`${STYLE_PREFIX} ${className || ''}`.trim()} {...props}>
+          <GraphView
+            ref={cy}
+            stylesheet={stylesheet}
+            elements={elements}
+            style={{ width: '700px', height: '700px', border: 'solid 2px gray' }}
```

Comment:
for sure this was done for testing only, as I thought this will be part of your CSS PR. I noticed when there is no styling the canvas doesn't exist and only the graph buttons appears with no canvas. At least we have to have a minWidth and minHeight for the canvas that at least contain the graph control buttons. So I changed this with minWidth and minHeight till your follow up PR can address this.